helix-term/src/ui/editor.rs

```diff
@@ -252,8 +254,16 @@ impl EditorView {
         theme: &Theme,
     ) {
         let editor_rulers = &editor.config().rulers;
+        let ruler_style = &editor.config().ruler_style;
+        let ruler_char = editor.config().ruler_char.to_string();
```

Comment:
```suggestion
        let ruler_char = editor.config().ruler_char; 
```
this creates an extra allocation on each render